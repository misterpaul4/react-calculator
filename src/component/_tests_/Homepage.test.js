import RRender from 'react-test-renderer';
import Homepage from '../Homepage';

it('renders the Homepage Page correctly without errors', () => {
  const tree = RRender.create(<Homepage />).toJSON();

  expect(tree).toMatchSnapshot();
});
