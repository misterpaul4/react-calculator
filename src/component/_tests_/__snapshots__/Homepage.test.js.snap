// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Homepage Page correctly without errors 1`] = `
Array [
  <h1>
    Welcome to our page!
  </h1>,
  <p>
    Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. IpsuLorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui o!cia deserunt mollit anim id est laborum. Ipsu
  </p>,
]
`;

exports[`renders the Quote Page correctly without errors 1`] = `
<p>
  Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding. â€“William Paul Thurston
</p>
`;
